rclip/utils/helpers.py

```diff
@@ -230,6 +234,11 @@ def read_raw_image_file(path: str):
   return Image.fromarray(np.array(rgb))
 
 
+def compute_image_hash(image: Image.Image) -> str:
+  """Compute perceptual hash for an image using pHash algorithm."""
+  return str(imagehash.phash(image))
```

Comment:
I wonder how likely this perceptual hashing algorithm is to match similar but different images. I would love rclip to detect moves of exactly same files, but if the files are different (different crops, etc), I think it's ok to just reindex them.

What do you think?